.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,45 @@
+name: Release drafter
+
+on:
+  push:
+    tags:
+      - "*"
```

Comment:
From what I can see this will create a release issue for every push/merge to any branch.

Is that what's intended?  Seems we'll have a lot of issue spam, most of which will be "no" clicks.

This would create an issue on PR merge, plus when the backport workflow pushed to the backport branch it would create another identical issue for that temporary branch.   It might also create a release issue for every mend version bump PR.  I suspect we need some branch filtering at a minimum.

And should we not have a manual way to trigger this workflow with `on:` `workflow_dispatch:`

Reply:
This will create a `draft release` on GitHub with the artifacts attached that need to be published.
More information on how the end to end process works: https://github.com/opensearch-project/opensearch-build/issues/1234

Reply:
Ah, got it, I forgot tags are a git thing and was thinking github labels!